models/intercom__conversation_enhanced.sql

```diff
@@ -63,25 +63,43 @@ team as (
 ),
 {% endif %}
 
+latest_conversation_enriched as (
+    select 
+        conversation_id,
+        created_at as conversation_created_at,
+        updated_at as conversation_last_updated_at,
+        source_type as conversation_type,
+        source_delivered_as as conversation_initiated_type,
+        source_subject as conversation_subject,
+        case when (assignee_type is not null) then assignee_type else 'unassigned' end as conversation_assignee_type,
+        source_author_type as conversation_author_type,
+        state as conversation_state,
+        is_read,
+        waiting_since,
+        snoozed_until,
+        sla_name,
+        sla_status,
+        conversation_rating_value as conversation_rating,
+        conversation_rating_remark as conversation_remark
+
+        {{ fivetran_utils.fill_pass_through_columns('intercom__conversation_history_pass_through_columns') }}
+
+    from latest_conversation
+),
+
 --Enriches the latest conversation model with data from conversation_part_events, conversation_string_aggregates, and conversation_tags_aggregate
-enriched as ( 
+enriched_final as ( 
```

Comment:
Was there a need to change the cte name? No preference to the update but just want to confirm

Reply:
No it wasn't necessary! I was just trying to tell a story that it was the last enriched CTE and to make it obvious the structure changed from before.

Reply:
Thanks for the clarification!